python/using_pyairtable/README.md

```diff
@@ -21,8 +21,6 @@ The software made available from this repository is not supported by Formagrid I
 - [`example.py`](example.py) is the main code file which is executed when `python3 example.py` is run. At a high level, it performs the following:
   - Loads dependencies and configuration variables
   - Defines a sample `inputRecords` array which should be modified to reference an external data source
-  - Retrieves all existing records in the Airtable base and creates a mapping of the unqiue field's value to the existing record ID for later updating
-  - Loops through each record from `inputRecords` array and determines if an existing record should be updated or a new one should be created
   - In chunks of 10, updates existing and creates new records
 - [`.env.example`](.env.example) is an example file template to follow for your own `.env` file. The environment variables supported are:
   - `AIRTABLE_API_KEY` - [your Airtable API key](https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-); it will always start with `key`
```

Comment:
Encourage use of PATs (but keep the env variable name the same)

```suggestion
  - `AIRTABLE_API_KEY` - [your Airtable personal access token](https://airtable.com/developers/web/guides/personal-access-tokens); it will always start with `pat`
```